src/ifElse.test.js

```diff
@@ -1,11 +1,57 @@
 'use strict';
 
 describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
+  const { ifElse } = require('./ifElse');
 
-  it('should ', () => {
+  it('should call the second callback when condition returns false', () => {
+    let conditionCalled = false;
+    let firstCallbackCalled = false;
+    let secondCallbackCalled = false;
 
+    const condition = () => {
+      conditionCalled = true;
+
+      return false;
+    };
+
+    const firstCallback = () => {
+      firstCallbackCalled = true;
+    };
+
+    const secondCallback = () => {
+      secondCallbackCalled = true;
+    };
+
+    ifElse(condition, firstCallback, secondCallback);
```

Comment:
You are not checking the return value of the 'ifElse' function. It's good to check if the function returns what we expect it to return.